api-server/src/server/utils/middleware.js

```diff
@@ -106,3 +106,36 @@ export const createValidatorErrorHandler =
 
     return next();
   };
+
+export async function ifValidWebhookURL(req, res, next) {
+  const standardErrorMessage = {
+    type: 'danger',
+    message: 'flash.wrong-updating'
+  };
+  const {
+    user: { username },
+    body: { webhook, webhookSecret }
+  } = req;
+
+  try {
+    await triggerWebhook(webhook, webhookSecret, username);
+    next();
+  } catch (error) {
```

Comment:
This error is forgotten about. I would send it to the client, and make the client error more specific that the sent resource did not successfully respond.